ui/push-health/Navigation.jsx

```diff
@@ -1,23 +1,47 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Navbar } from 'reactstrap';
+import { Navbar, Tooltip } from 'reactstrap';
 
 import LogoMenu from '../shared/LogoMenu';
 import Login from '../shared/auth/Login';
 
 export default class Navigation extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      explainerOpen: false,
+    };
+  }
+
+  toggleExplainer = () => {
+    const { explainerOpen } = this.state;
+
+    this.setState({ explainerOpen: !explainerOpen });
+  };
+
   render() {
+    const { explainerOpen } = this.state;
     const { user, setUser } = this.props;
 
     return (
       <Navbar dark color="dark">
         <LogoMenu menuText="Push Health" />
-        <span
-          title="This data is for UI prototyping purposes only"
-          className="text-white"
-        >
+        <span id="prototype" className="text-white">
           [---PROTOTYPE---]
         </span>
+        <Tooltip
```

Comment:
I've created a `SimpleTooltip` component that you could probably use here instead - it's in the shared file.

Reply:
Nice!  I'll use it.  :)